### PR TITLE
UTF-8 support for Config class.

### PR DESCRIPTION
**Pull request by sarveshkaushal**
Looked at another TODO item in Config file. Please take a look at code change.

### CODE REVIEW

File: util/src/main/java/io/kubernetes/client/util/Config.java [hunk 1]
```diff
@@ -74,13 +76,17 @@ public static ApiClient fromToken(String url, String token, boolean validateSSL)
...
 
   public static ApiClient fromConfig(String fileName) throws IOException {
-    KubeConfig config = KubeConfig.loadKubeConfig(new FileReader(fileName)); // TODO UTF-8
+    KubeConfig config =
+        KubeConfig.loadKubeConfig(
+            new BufferedReader(
+                new InputStreamReader(
+                    new FileInputStream(fileName), StandardCharsets.UTF_8.name())));
     config.setFile(new File(fileName));
     return fromConfig(config);
```
Comment (brendandburns) [inline util/src/main/java/io/kubernetes/client/util/Config.java]: We should really close the `FileInputStream` here.  Perhaps:

```java
try (FileInputStream fi = new FIleInputStream(fileName)) {
   KubeConfig config = KubeConfig.loadKubeConfig(...);
   return  fromConfig(config);
}
```

**Comment by brendandburns** [comment]
Thanks for the PR, one small improvement requested.

File: util/src/main/java/io/kubernetes/client/util/Config.java (hunk 1 above)
Comment (sarveshkaushal) [reply util/src/main/java/io/kubernetes/client/util/Config.java]: Yes try-with-resources is correct way of doing this here. Thanks for your input. I will change it.

**Comment by sarveshkaushal** [comment]
@brendanburns - I have added the changes as per your input. Thanks for pointing it out. Please take a look.

**Comment by yue9944882** [comment]
the flakiness is connected w/ https://github.com/tomakehurst/wiremock/issues/515. sadly #745 doesnt actually solve that

**Comment by sarveshkaushal** [comment]
Closing this. Will open again with latest changes from master.

**Pull request closed without merging**